faq/faq.rst

```diff
@@ -72,3 +72,10 @@ How does citus support count(distinct) queries?
 -----------------------------------------------
 
 Citus can push down count(distinct) entirely down to the worker nodes in certain situations (for example if the distinct is on the distribution column or is grouped by the distribution column in hash-partitioned tables). In other situations, Citus uses the HyperLogLog extension to compute approximate distincts. You can read more details on how to enable approximate :ref:`count_distinct`.
+
+In which situations may I add a uniqueness constraint to a distributed table?
+-----------------------------------------------------------------------------
+
+Citus is able to enforce a primary key or uniqueness constraint on columns only when one of the columns is the distribution column. In particular this means that if a single column constitutes the primary key then it has to be the distribution key as well.
```

Comment:
Minor, slight re-phrasing. Feel free to incorporate or ignore.

"Citus is able to enforce a primary key or uniqueness constraint only when the primary key constraint contains the distribution column. In particular this means that if a single column constitutes the primary key then it has to be the distribution key as well."